Competitive Coding/Math/Catalan_Numbers/README.md

```diff
@@ -0,0 +1,21 @@
+The 2 different programs in this folder contain the same implementation!
+The programs basically are a python implementation of the Catalan Numbers.
+They're similar to Fibonacci (very slightly)
+
+The 2 different implementations vary only in terms of time taken!<br>
+Basically we can implement in 3 different ways!<br>
+
+(1) Recursively <br>
```

Comment:
instead of numbering use `*`.

And write the time complexities here only like this

* Recursive  -exponential